src/app/genes/gene-search/gene-search.component.ts

```diff
@@ -46,20 +50,11 @@ export class GeneSearchComponent implements OnInit {
     initQueryField() {
         this.queryField.valueChanges
             .pipe(
-                debounceTime(300),
+                debounceTime(500),
                 distinctUntilChanged(),
                 switchMap((query) => {
                     this.results = [];
-
-                    if (query) {
-                        this.currentQuery = query;
-                        this.isSearching = true;
-                        return this.search(query);
-                    } else {
-                        this.currentQuery = '';
```

Comment:
Why is this no longer needed?  Are there cases when query is not set and a reset is necessary?

Reply:
Tagging @mattcgenui

Reply:
Sorry @jay-hodgson, I missed this feedback. This code was move to line 114/115: 
https://github.com/Sage-Bionetworks/Agora/pull/938/files#diff-1304225f209da309cb2ce390fc0e905f8af131af9b36f957ea3dbc53b2369e38R114